api/room/schema_query.py

```diff
@@ -280,7 +280,8 @@ def resolve_all_rooms(self, info, **kwargs):
     def resolve_all_remote_rooms(self, info, return_all=None):
         page_token = None
         filter = map_remote_room_location_to_filter()
-        location = 'all' if return_all else get_user_from_db().location
+        location = get_user_from_db().location
+        if not filter.get(location) or return_all: location = 'all'      
```

Comment:
@MCFrank16 put the assignment on a separate line to allow for easy reading of your code.

Reply:
ok... let me do it